README.md

```diff
@@ -1,13 +1,34 @@
 # snowbox [![Build Status](https://travis-ci.com/algotech/snowbox.svg?branch=master)](https://travis-ci.com/algotech/snowbox)[![codecov](https://codecov.io/gh/algotech/snowbox/branch/master/graph/badge.svg)](https://codecov.io/gh/algotech/snowbox)
 
-## Install
+Opinionated Redux abstractions for faster development.
 
-Install from the NPM repository using yarn or npm:
+## Motivation
```

Comment:
@lgnstn Shouldn't this pe `Problem` instead of `Motivation`?

Reply:
I don't have a strong feeling for either of them. @ssergiuss?